versions/build.ver

```diff
@@ -23,12 +24,14 @@ export bacio_ver=2.4.1
 export w3nco_ver=2.4.1
 export nemsio_ver=2.5.2
 export sigio_ver=2.3.2
-export w3emc_ver=2.7.3
+export w3emc_ver=2.9.2
```

Comment:
I will just note here that; `w3emc 2.9.2` subsumes `w3nco`; i.e. we no longer need `w3nco`.  This however will likely break codes that build with both.  Something to note for next iteration and tell all our developers to remove `w3nco` dependency if using `w3emc >= 2.9.2`.

Reply:
Noted, thanks!